LUSDChickenBonds/src/BLUSDLPZap.sol

```diff
@@ -96,4 +96,54 @@ contract BLUSDLPZap {
 
         return bLUSDLUSD3CRVTokens;
     }
+
+    function removeLiquidityBalanced(uint256 _lpAmount, uint256 _minBLUSD, uint256 _minLUSD) external {
+        // pull LP tokens
+        bLUSDLUSD3CRVLPToken.transferFrom(msg.sender, address(this), _lpAmount);
+
+        // All those balances should be zero, but just in case:
+        uint256 initialLUSD3CRVBalance = lusd3CRVPool.balanceOf(address(this));
+        uint256 initialBLUSDBalance = bLUSDToken.balanceOf(address(this));
+
+        // withdraw bLUSD/LUSD-3CRV
+        bLUSDLUSD3CRVLPToken.approve(address(bLUSDLUSD3CRVPool), _lpAmount);
+        bLUSDLUSD3CRVPool.remove_liquidity(_lpAmount, [_minBLUSD, 0], false, address(this));
```

Comment:
Do we also need a minimum for the received LUSDCRV-f token quantity here? I guess this pool could be more easily manipulated than larger Curve pools (especially in early days), but haven't thought about attacks / extractable profit for it